app.rb

```diff
@@ -0,0 +1,45 @@
+@alphabets = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z'
+}
+
+def decode_char(string)
+  @alphabets[string]
+end
+
+def decode_word(word)
+  word.split(/ /).map { |letter| decode_morse letter }.join
+end
+
+def decode_sentence(sentence)
+  sentence.split(/  /).map { |word| decode_word word }.join(' ')
+end
+
+puts decode_char('.-')
+puts decode_word('-- -.--')
+puts decode('-- -.--   -. .- -- .')
+puts decode('.- -... --- -..- ..-. ..- .-.. .-.. --- ..-. .-. ..- -... .. . ...')
```

Comment:
- [ ] I think it would be better if you also update the above lines of code by using the correct methods names. Right now it raisesan `NoMethodError` error because you don't have a method named `decode`. You should change them to these:
```ruby
puts decode_sentence('-- -.--   -. .- -- .')
puts decode_sentence(".-   -... --- -..-   ..-. ..- .-.. .-..    --- ..-.    .-. ..- -... .. . ...")
```